docs/deployment.md

```diff
@@ -0,0 +1,91 @@
+
+This document details the process we use to deploy Page Shot to our stage and production environments.
+
+## Overview of schedule ##
+
+(you can read more below)
+
+- Thursday at 12pm PST the train is cut, all code is in. Style tweaks can be made after this point.
+- Friday at 9am PST we tag and start pushing to stage, then send email out.
+- Monday at regular stand-up we review anything that Softvision found and fix it.
+- Wednesday at ~8am PST we push and Softvision verifies.
```

Comment:
I only think we should push this off to Wednesday (instead of Tuesday) if SoftVision thinks it needs that time.  I don't think it will be an issue for ops.